getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Configure Image deployment by digest
```

Comment:
Deploy image by digest

(I realize this about how to configure this feature, but I think the title implies the content will tell you how to configure this feature).